srcpkgs/xfce4-panel-profiles/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'xfce4-panel-profiles'
+pkgname=xfce4-panel-profiles
+version=1.0.14
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --python=python3"
+hostmakedepends="intltool tar"
+depends="xfce4-panel python3-gobject"
+short_desc="Simple application to manage Xfce panel layouts"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
should not be orphaned

Reply:
It's marked orphaned as I don't want to be a maintainer, but it's okay to fill my profile if you said that.

Reply:
if you don't want to be maintainer, why make the PR?

Reply:
Because I'm indeed in need of it, and making such contribution does not cost me anything. Moreover, this package _is_ quite useful for Xfce users IMO.

Tbh I'm fine with being a maintainer, you can either close/merge this.